actors/abi/sector.go

```diff
@@ -207,12 +207,16 @@ func (p RegisteredSealProof) RegisteredWindowPoStProof() (RegisteredPoStProof, e
 	}
 }
 
+
+// PARAM_SPEC
+// The maximum duration (activation to expiration) of a sector sealed with SDR.
+// Motivation: This gurantees that SDR is secure in the cost model for WindowPoSt and in the time model for WinningPoSt
+// This is based on estimation of hardware latency improvement and hardware and software cost reduction.
+const SectorMaximumLifetimeSDR = abi.ChainEpoch(builtin.EpochsInYear * 5)
```

Comment:
**Possible Change**: we should change this according to analysis here: https://github.com/filecoin-project/research-private/issues/242

ping @irenegia, @lucaniz

Reply:
Considering:

1. We are in the pragmatic model
2. Our Amax is ~5 (5.14 according to this doc which is the up to date reference [checked with @irenegia ])
3. HW cost is halving every 1 to 2 years [assumption]
We have that
- A conservative approach would suggest to take
`RegisteredSealProof.SectorMaximumLifetime` = 1y
- A more aggressive approach could go to
`RegisteredSealProof.SectorMaximumLifetime` = 2 y (edited) 

Everything which is above 2 years seems insecure, under the assumption in point 3.